bodhi/server/config.py

```diff
@@ -377,6 +377,12 @@ class BodhiConfig(dict):
         'koji_hub': {
             'value': 'https://koji.stg.fedoraproject.org/kojihub',
             'validator': str},
+        'koji_pending_signing_side_tag': {
```

Comment:
We have per release configuration settings for example (https://github.com/fedora-infra/bodhi/blob/develop/production.ini#L430).

I would feel better if this was something that we configure only for a release (rawhide in that case). That will give us less chances to break things :smile:

Reply:
Could I set default for those settings?

Reply:
It looks like I can set defaults for `rawhide.koji_pending_signing_side_tag`, so I will change it.

Reply:
Do we need defaults tho ? if nothing is configured then we should not do anything. What do you think ?

Reply:
I think it is better to have default value then checking, if the configuration exists every time you want to work with it.

Reply:
I don't have a strong opinion on that, but if we want a default then let's use something similar to the mandatory_days_in_testing. (ie use fedora.your_setting_here instead of rawhide).

See https://github.com/fedora-infra/bodhi/blob/1a48548d911566fb54a187f41bc74c61f4430572/bodhi/server/models.py#L861

I think longer term we might want to be able to create updates from side tag for more than rawhide.

Reply:
This looks good. I will add something similar for side tags.